recipes/dtaidistance/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dtaidistance" %}
+{% set version = "2.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9d9b5be57f1abdac504e271be2d1c9d10cdd7b5cf30b435dfba8b05090a531b6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cython
+    - numpy
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+  run:
+    - {{ pin_compatible('numpy') }}
+    - cython
```

Comment:
`cython` doesn't seem to be used at runtime:
```suggestion
```

Reply:
I also did not have `cython` at first listed here. However `pip check` failed and it seems to be listed in the requirements of the package: https://github.com/wannesm/dtaidistance/blob/v2.3.2/setup.py#L340

What do you recommend? Remove `pip check` or leave `cython` in run requirements?

Reply:
Personally, I'd prefer not to force a build tool (Cython) on all the users of the package. But I'll leave it up to you as recipe maintainer to way that against the benefits of `pip check`.

Reply:
I am of the same opinion, thanks for your feedback. Will discuss with the maintainer of the source repo when I add a conda-forge badge.